client/signup/main.jsx

```diff
@@ -709,7 +713,7 @@ class Signup extends React.Component {
 						shouldShowLoadingScreen={ this.state.shouldShowLoadingScreen }
 						isReskinned={ isReskinned }
 						rightComponent={
-							showProgressIndicator && (
+							showProgressIndicator( this.props.flowName ) && (
```

Comment:
Yeah, I also consider this before. But I think it could be a separate PR later to remove all of the flow name checking. Furthermore, maybe we can also make `reskinned_flows` be a flag named `isReskinned` in `flows-pure.js`?

Reply:
Yeah I'd like that too. It took me a while initially to find where `isReskinned` was set, because I really wasn't expecting an array in the config file. It might have been useful during the initial roll out I guess.